### ISSUE TITLE
Colomoto Slides

### ISSUE THREAD

**Issue by tlaguna** [migp11/ECCB2020-T05#1]
Hi, I was wondering if the slides from the Colomoto part of the tutorial will be uploaded... and thanks for the great work in the tutorial!

**Comment by migp11** [comment]
Hi,

Which slides are you referring exactly?
All the slides have been uploaded to:
https://github.com/migp11/ECCB2020-T05/tree/master/slides

except by Pedro's, who did no use any presentation but a notebook.
[https://github.com/migp11/ECCB2020-T05/tree/master/notebooks/attractors](https://github.com/migp11/ECCB2020-T05/tree/master/notebooks/attractors)

Inside Pedro's notebook you will find links to colomoto project.
Best
Miguel

**Comment by tlaguna** [comment]
I think I remember Pedro showing some slides, am I mistaken? 

He was showing some information about the method in general and at the end opened the notebook and ran it. Maybe they were not slides? What I am sure is that it was not contained in the notebook...

**Comment by ptgm** [comment]
Hi,
Indeed, I did not show any specific slides. Just a few supporting ones to illustrate the passage from a set of genes with a topological network, then with some associated boolean functions, which would then induce some dynamics (I talked a bit about the difference between synchronous vs asynchronous).
This introduction of dynamics, was then to be able to use the tutorial, since we may be interested in searching for attractors, as well as if they are reachable from a given (set of) initial conditions.
Best,
Pedro
